src/System.Security.Cryptography.X509Certificates/tests/CollectionImportTests.cs

```diff
@@ -309,6 +310,26 @@ public static void ImportPkcs12File_Chain_VerifyContents(X509KeyStorageFlags key
             }
         }
 
+#if netcoreapp11
+        [Fact]
+        [PlatformSpecific(TestPlatforms.OSX)]
+        public static void EphemeralKeySet_OSX()
+        {
+            // EphemeralKeySet fails when loading a PFX, and is ignored otherwise.
+            using (ImportedCollection coll = Cert.Import(TestData.Pkcs7ChainDerBytes, null, X509KeyStorageFlags.EphemeralKeySet))
+            {
+                Assert.Equal(3, coll.Collection.Count);
+            }
+
+            Assert.Throws<PlatformNotSupportedException>(
+                () =>
+                {
+                    using (Cert.Import(TestData.EmptyPfx, null, X509KeyStorageFlags.EphemeralKeySet))
```

Comment:
What's throwing the PNSE?  Assuming it's the call to Import, there's no need for the using.